tests/jaxpr_effects_test.py

```diff
@@ -19,14 +19,14 @@
 import jax
 import jax.numpy as jnp
 from jax import lax
-from jax.experimental import maps
 from jax.experimental import pjit
 from jax._src import ad_checkpoint
 from jax._src import dispatch
 from jax._src import config
 from jax._src import core
 from jax._src import effects
 from jax._src import linear_util as lu
+from jax._src import maps
```

Comment:
Can this be from jax._src.maps import xmap?

Reply:
Done